core/src/main/scala/dagr/core/config/Configuration.scala

```diff
@@ -218,8 +218,8 @@ private[config] trait ConfigurationLike extends LazyLogging {
     }
   }
 
-  def commandLineName: String = {
-    configure(Configuration.Keys.CommandLineName, "Dagr")
+  def commandLineName(default: String = "dagr"): String = {
```

Comment:
It's confusing to me to have a parameter named `default` since that's a keyword in Java.  I thought for a minute this had special meaning here too.